docs-gen/content/rule_set/instances.md

```diff
@@ -35,6 +37,19 @@ interpreted by the tools.
    four position instances will be created for each of the 'Left' and 'Right' instances,
    resulting into a total number of 8 instances.
 
+### How can I exclude child-nodes from instantiation?
+
+Often it makes sense to instantiate all child-nodes of a branch.
+But there are cases, when nodes are linked more the general concept of
+a branch, but not to the single instance. This could be the `DoorCount`,
+which would rather be `Door.Count`, `WheelDiamater`, which is rather linked
+to an axle rather than the wheel itself or `Brake.FluidLevel` which is not
+measured for a single break, but rather a system indication. 
+
+To exclude a child-node from the instantiation of the *direct* parent node, set the
+keyword `instantiate` to `false` (`true` by default). Please check the following
```

Comment:
In the vss-tools PR we use `False` in the test specification. That does not seem to matter, our Python tooling is forgiving and accepts many synonyms, like `No` and `Off`, but I would suggest that we are consistent in the repos. I.e. either use `False` here, or change to `false` in the test case